api/db-cdr/changelog-schema/db.changelog-master.xml

```diff
@@ -9,6 +9,5 @@
     <runningAs username="liquibase"/>
   </preConditions>
   <include file="changelog-schema/db.changelog-v2-1.xml"/>
-  <include file="changelog-schema/db.changelog-v2-2-cb_menu.xml"/>
```

Comment:
The proper way to handle this is the add a new changelog file that deletes the cb_menu table